helm/hpcc/values.yaml

```diff
@@ -29,7 +29,13 @@ global:
   # If not specified, the first esp component that exposes eclservices application is assumed.
   # Can also be overridden locally in individual components
   ## defaultEsp: eclservices
-  
+
+  # The locale used for internationalization
+  locale:
```

Comment:
What was the reason for moving to a new section? I can't see the other entries being used.

Reply:
Please see previous comment regarding making locale more generic.